.github/workflows/ci.yml

```diff
@@ -1,53 +1,32 @@
 name: CI
 
-on: [push, pull_request]
+on:
+  push:
+    branches: [main]
```

Comment:
do we only want this to run on pushes to master and not when pull requests are opened?

Reply:
Done